test/openfisca/mapping/reverse.coffee

```diff
@@ -27,6 +27,13 @@ describe 'Reverse mapping', ->
     bourse_college: '2014-11': 1
     bourse_lycee: '2014-11': 1
     paris_logement_familles: '2014-11': 1
+    paris_forfait_famille: '2014-11': 1
+    paris_logement_psol: '2014-11': 1
+    paris_logement: '2014-11': 1
+    paris_logement_plfm: '2014-11': 1
+    paris_logement_aspeh: '2014-11': 1
+    paris_energie_famille: '2014-11': 1
+    paris_complement_sante: '2014-11': 1
```

Comment:
This list is getting long... -_-
Thought about automating it but I don't know coffee script.